Athos/TFEzPCLibrary/Library32_common.ezpc

```diff
@@ -24,24 +24,32 @@ SOFTWARE.
 
 (**************************)
 (* TODO : the 2nd arg should be broadcasted *)
-def void MatAddBroadCast2(int32_pl s1, int32_pl s2, int32_al[s1][s2] A, int32_al[s2] B, int32_al[s1][s2] outArr){
+def void MatAddBroadCast2(int32_pl a1, int32_pl a2, int32_pl b1, int32_pl s1, int32_pl s2, int32_al[s1][s2] A, int32_al[s2] B, int32_al[s1][s2] outArr){

```

Comment:
I didn't get this change - a1, a2, b1 - are not getting used anywhere as far as i can see

Reply:
Oh, is this to make sure you add args for each array variable sizes? And you don't end up using it because for eg a1=s1 and you use s1 instead?

Reply:
Yes, probably. I think @bhatuzdaname made these changes. Can you take a look a look if this is correct?